tests/config_test.py

```diff
@@ -1630,6 +1630,145 @@ def teardown_class(cls):
         print("TEARDOWN")
 
 
+class TestConfigWarmRestart(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        import config.main
+        importlib.reload(config.main)
+
+    @patch("validated_config_db_connector.device_info.is_yang_config_validation_enabled", mock.Mock(return_value=True))
+    @patch("config.validated_config_db_connector.ValidatedConfigDBConnector.validated_mod_entry", mock.Mock(side_effect=ValueError))
```

Comment:
Could this unit testing be done specifically in yang models testing, instead of in sonic-utilities? 

I chose to throw ValueError because I think YANG testing can be separate. If we would like to use YANG validation itself directly in sonic-utilities, it may be better to wait until after GCU code is refactored (to make YANG validation process more efficient) so that sonic-utilities testing performance is not heavily impacted. If we use YANG itself directly in sonic-utilities tests, the testing time will skyrocket.